### PR TITLE
Update Admissions contact info on grad.admissions.uiowa.edu

### LINKED ISSUE #6489: Update admissions contact info on Grad Admissions
# Story

As a content editor for grad.admissions.uiowa.edu, I would like to have our contact information on Areas of Study be updated to reflect our new location so that we do not have to manually update hundreds of nodes.

# Additional Info

The old info:
Office of Admissions
The University of Iowa
108 Calvin Hall
Iowa City, IA 52242-1396

The new info:
Enrollment Management
The University of Iowa
2900 University Capitol Centre
201 S. Clinton St.
Iowa City, IA 52242

Scope only to field_area_of_study_contact

# Proposed Solution

- Update hook or database find/replace to update contact info

### PR DESCRIPTION
**Pull request by pyrello**
Resolves #6489

# How to test

```
ddev blt ds --site grad.admissions.uiowa.edu
```
- Ensure the update hook runs without issue.
- The update hook will print out a list of node IDs that were updated. Spot check the results using this list.
- Ensure that a new revision was created.

### CODE REVIEW

**Comment by bspeare** [comment]
Testing this.

File: docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/grad_admissions_core.install [hunk 1]
```diff
@@ -66,3 +66,55 @@ function grad_admissions_core_update_9001() {
...
+
+/**
+ * Update contact info strings.
+ */
+function grad_admissions_core_update_9002(&$sandbox) {
+  $contact_field_name = 'field_area_of_study_contact';
+  $updated = [];
+  $conversion_map = [
+    'Office of Admissions' => 'Enrollment Management',
+    '108 Calvin Hall' => '2900 University Capitol Centre<br />\n201 S. Clinton St.',
```
Comment (bspeare) [inline docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/grad_admissions_core.install]: Can we remove `\n` here?

File: docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/grad_admissions_core.install [hunk 2]
```diff
@@ -66,3 +66,55 @@ function grad_admissions_core_update_9001() {
...
+    '52242-1396' => '52242',
+  ];
+
+  // @todo Check if there are instances of 108 Calvin Hall that are used
```
Comment (bspeare) [inline docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/grad_admissions_core.install]: Does this `@todo` need to be here?

Comment (pyrello) [reply docroot/sites/grad.admissions.uiowa.edu/modules/grad_admissions_core/grad_admissions_core.install]: Nope

**Review by bspeare — APPROVED** [review]
This worked great as long as you are good with the changes I pushed up.

**Pull request merged**